src/panels/lovelace/cards/hui-weather-forecast-card.js

```diff
@@ -2,7 +2,22 @@ import "../../../cards/ha-weather-card";
 
 import LegacyWrapperCard from "./hui-legacy-wrapper-card";
 
+// should be interface when converted to TS
+export const Config = {
+  entity: "",
+  name: "",
+};
+
 class HuiWeatherForecastCard extends LegacyWrapperCard {
+  static async getConfigElement() {
+    await import("../editor/config-elements/hui-weather-forecast-card-editor");
```

Comment:
Add a chunk name. See #2278

Reply:
Okay, I'll add this to my current open editor PRs and then go back and tackle that referenced issue next with some other cleanup PRs.